### DISCUSSION TITLE
Is there a hard limit to the number of watch folders?

### DISCUSSION THREAD

**Discussion by StevenDTX** [jlesage/docker-handbrake#338] [Q&A] [answered]
I set up a few watch folders, and in my testing it all worked fine.  I have 15 different Handbrake profiles that I regularly use, so I set up 15 watch folders.

However, nothing works past /watch5.  I can drop a file in /watch1-5, and it converts as expected. If I drop a file in /watch6-15, the file is never detected.

Here is my docker config:
https://pastebin.com/6K0m5DCa

Thanks!

**Comment by jlesage** [comment, marked as answer]
You can add the variable `AUTOMATED_CONVERSION_MAX_WATCH_FOLDERS` to set the maximum number of watch folders.
https://github.com/jlesage/docker-handbrake?tab=readme-ov-file#environment-variables

**Comment by StevenDTX** [comment]
Wow! I totally missed that setting. That fixed my issue. 

Thank you very much!!

**Reply by jlesage** [reply]
You are welcome!
